tests/PHPStan/Type/ObjectTypeTest.php

```diff
@@ -418,6 +419,16 @@ public function dataIsSuperTypeOf(): array
 				new ObjectType(ExtendsThrowable::class),
 				TrinaryLogic::createMaybe(),
 			],
+			59 => [
+				new ObjectType(DateTime::class),
+				new ObjectType(ConstantNumericComparisonTypeTrait::class),
```

Comment:
pretty sure php has no built-in traits so i used one that phpstan has, i feel like theres a better way to do this though